.github/workflows/e2e.yml

```diff
@@ -0,0 +1,40 @@
+name: Soroban Tools e2e
+
+on:
+  push:
+    branches: [main, release/**]
+  pull_request:
+
+jobs:
+
+  integration:
+    name: System tests
+    strategy:
+      matrix:
+        scenario-filter: ["^TestDappDevelop$/^.*$"]
+    runs-on: ubuntu-latest
+    env:
+      SYSTEM_TEST_DOCKER_REPO_TAG: "sreuland/system-test:latest" 
+      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "main"
+      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_REPO: "https://github.com/stellar/soroban-examples.git"
+      SYSTEM_TEST_RUST_TOOLCHAIN_VERSION: "1.65.0"
+      SYSTEM_TEST_CORE_DEBIAN_VERSION: "19.5.1-1137.b3a6bc281.focal~soroban"
```

Comment:
I think that it's good for now, but for the long run, we need to derive the core version from the one in the go mono repository. To do that:
1. open the go.mod file and figure out the go repository hash.
2. checkout whatever file in the go repo that contain the core dependency
3. populate the ENV vars.

Reply:
created https://github.com/stellar/soroban-tools/issues/328 to track this